independent-projects/tools/common/src/test/resources/templates/README.gradle.ftl

```diff
@@ -0,0 +1,40 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+More detail about using maven for quarkus here: https://quarkus.io/guides/gradle-tooling
```

Comment:
If we really need these files, let's make them only `# ${project_artifactId} project` and remove all the rest of the content so that they don't end up being outdated.

Reply:
OK, I reduced the file content to the "build in dev mode" command line.
So in test we are able to check that we chosen the right README template (the gradle or the maven one).